java/rocksjni/portal.h

```diff
@@ -5010,84 +5010,96 @@ class TickerTypeJni {
         return -0x07;
       case ROCKSDB_NAMESPACE::Tickers::BLOB_DB_FIFO_BYTES_EVICTED:
         return -0x08;
-      case ROCKSDB_NAMESPACE::Tickers::TXN_PREPARE_MUTEX_OVERHEAD:
+      case ROCKSDB_NAMESPACE::Tickers::BLOB_DB_CACHE_MISS:
         return -0x09;
-      case ROCKSDB_NAMESPACE::Tickers::TXN_OLD_COMMIT_MAP_MUTEX_OVERHEAD:
+      case ROCKSDB_NAMESPACE::Tickers::BLOB_DB_CACHE_HIT:
         return -0x0A;
-      case ROCKSDB_NAMESPACE::Tickers::TXN_DUPLICATE_KEY_OVERHEAD:
+      case ROCKSDB_NAMESPACE::Tickers::BLOB_DB_CACHE_ADD:
         return -0x0B;
-      case ROCKSDB_NAMESPACE::Tickers::TXN_SNAPSHOT_MUTEX_OVERHEAD:
+      case ROCKSDB_NAMESPACE::Tickers::BLOB_DB_CACHE_ADD_FAILURES:
         return -0x0C;
-      case ROCKSDB_NAMESPACE::Tickers::TXN_GET_TRY_AGAIN:
+      case ROCKSDB_NAMESPACE::Tickers::BLOB_DB_CACHE_BYTES_READ:
         return -0x0D;
-      case ROCKSDB_NAMESPACE::Tickers::FILES_MARKED_TRASH:
+      case ROCKSDB_NAMESPACE::Tickers::BLOB_DB_CACHE_BYTES_WRITE:
         return -0x0E;
+      case ROCKSDB_NAMESPACE::Tickers::TXN_PREPARE_MUTEX_OVERHEAD:
```

Comment:
Will this be a problem for applications that have been using the old ticker values?

Reply:
That's interesting. But I think I modified Java type's offset. https://github.com/facebook/rocksdb/pull/10203/files#diff-f0c2cf9706079955f87599b91f4041e3b9f899d331fe4368ae49963edc91ad99R698-R716

This is too restricted if we can only add new tickers at the tail. @ltamasi

Reply:
Also cc @adamretter since this is related to RocksJava.

Reply:
To me it definitely seems safer to add them after the existing ones; it's OK to not have all BlobDB related ones together.

Reply:
as he did not rely on this. I'll add them after the current ones as you suggested.

Reply:
done